SSA/Projects/InstCombine/ForLean.lean

```diff
@@ -590,3 +590,30 @@ theorem Bool.xor_decide (p q : Prop) [dp : Decidable p] [Decidable q] :
     (decide p).xor (decide q) = decide (p ≠ q) := by
   cases' dp with pt pt
   <;> simp [pt]
+
```

Comment:
When you move the unrelated Boolean statements to `ForLean.lean`, you could also drop the `Bool.` prefix and create a section instead.

Reply:
I implemented this.